terraform/environments/production/dns.tf

```diff
@@ -145,7 +145,7 @@ resource "google_dns_record_set" "sendgrid-return-1" {
   managed_zone = module.google-cloud-dns.zone_name
 
   type    = "CNAME"
-  name    = "em8227.${module.google-cloud-dns.dns_name}"
+  name    = "em7493.${module.google-cloud-dns.dns_name}"
```

Comment:
I believe this was the main issue

Reply:
So, funny enough, the `em8227` validation records are already in Google Cloud DNS.  With the exception of the `_dmarc` record value.  I had meant to discuss with you whether those records were still valid or if I needed to wipe those and start fresh.

If we're ok using the `em8227` entry in Sendgrid, then I think all that needs to be done is hitting the 'Verify' button

Reply:
Sure. I thought I hit "verify" and it showed me a screen with this cname to set, but I could have rushed through.

Reply:
I think you may have hit the "verify" for the `em7493` record.  I just hit "verify" for the `em8227` record and it's now verified.

Reply:
Awesome. I'll update this PR then revert

Reply:
Updated. There's still the reverse DNS change we should probably merge I think so that we are using our dedicated IP properly.

Reply:
This fixed the delivery issues btw.